pkg/reporting/facts.go

```diff
@@ -1,13 +1,55 @@
-package factsmodel
+package reporting
 
 import (
 	"context"
 	"fmt"
+	"reflect"
+
 	"github.com/appuio/appuio-cloud-reporting/pkg/db"
 	"github.com/jmoiron/sqlx"
-	"reflect"
+	ctrl "sigs.k8s.io/controller-runtime"
 )
 
+func FactByRecord(ctx context.Context, tx *sqlx.Tx, dt *db.DateTime, record Record) (*db.Fact, error) {
+	query, err := GetQueryByName(ctx, tx, record.QueryName)
```

Comment:
AFAICT this function is only used in integration tests, right?  In that case I'd prefer to move it to the test suite to reduce API surface.

Reply:
yeah I was wondering where to put it and put it into reporting package to have it at the same location as other Facts related code. I agree about reducing API surface and move it to `test` package :+1: